src/main/scala/org/broadinstitute/dsde/vault/VaultConfig.scala

```diff
@@ -63,7 +63,7 @@ object VaultConfig {
 
     def uBamResolvePath(id: String) = dm.getString("ubamsResolvePath").format(id)
     def uBamResolveUrl(id: String) = server + uBamResolvePath(id)
-
+    def uBamResolveListUrl() = ubamsUrl
```

Comment:
[IMHO](http://www.urbandictionary.com/define.php?term=IMHO) this should be a `lazy val`, evaluated once like the other values, not a `def` pointing to a `lazy val`.